django/db/models/lookups.py

```diff
@@ -309,7 +309,7 @@ def batch_process_rhs(self, compiler, connection, rhs=None):
         return sql, tuple(params)
 
 
-class PostgresOperatorLookup(FieldGetDbPrepValueMixin, Lookup):
+class PostgresOperatorLookup(Lookup):
```

Comment:
Not every `PostgresOperatorLookup` subclass needs to have its `rhs` go through a `lhs.get_db_prep`.

For example, lookups with a mismatching rhs and lhs type. For example, `HasKeyLookup` which has a `(JSONField, TextField)` since `jsonb ? text → boolean` and not `jsonb ? jsonb`